react-app/src/ExerciseOverview/index.js

```diff
@@ -15,6 +16,7 @@ export default function ExerciseOverview({ route }) {
   const [selectedStep, setSelectedStep] = useState(1)
   const exercises = { 1: STRETCHING_EXERCISE_DATA, 2: CORE_EXERCISE_DATA }
   const exercise = exercises[id]
+  const summary = DUMMY_FUNCTION_DATA[0].summary
```

Comment:
You can use exercise.summary instead

Reply:
You can use `DUMMY_FUNCTION_DATA.filter(exerciese => exercise.id === id ).summary` instead

Reply:
it seem not to work correctly.
so i find another way 
`DUMMY_FUNCTION_DATA[id - 1].summary`